tests/test_acasclient.py

```diff
@@ -437,6 +437,16 @@ def create_and_connect_backdoor_user(self, username = None, password = None, pre
         user_client = acasclient.client(user_creds)
         return user_client
 
+    def verify_file_and_content_equal(self, content, file):
+        """ Compare the content to the file contents"""
+        if isinstance(content, bytes):
+            with open(file, 'rb') as f:
+                upload_bytes = f.read()
+                self.assertEqual(content, upload_bytes)
+        else:
+            with open(file, 'r') as f:
+                self.assertEqual(content, f.read())
```

Comment:
`nit:` You can also consider writing it as,
```python
mode = 'rb' if isinstance(content, bytes) else 'r'
with open(file, mode) as f:
    sefl.assertEqual(content, f.read())
```

Reply:
Thanks, thats a nice simplification